### PR TITLE
Add method for more interpolated Laserscans

### PR DESCRIPTION
**Pull request by Spyrko**
New method which converts the laser scan into an interpolated pointcloud according to the given transformation, the startTime, the angular_resolution and the rotation speed of the laserbeam. Acts just like convertScanToPointCloud() but considers the motion of the laser scanner.

### CODE REVIEW

**Comment by jmachowinski** [comment]
looks good, ready to merge from my side

File: base/samples/LaserScan.hpp [hunk 1]
```diff
@@ -130,6 +132,53 @@ namespace base { namespace samples {
...
+        //moving the check for skip_invalid_points
+        //out of the loop speeds up the execution 
+        //time by ~25 %
+        if(!skip_invalid_points)
+        {
+            for(unsigned int i = 0; i < ranges.size(); i++) {
+                base::samples::RigidBodyState bodyState;
+                Eigen::Vector3d point;
+                if(getPointFromScanBeamXForward(i, point) && transformation.get(startTime + base::Time::fromSeconds((start_angle / angular_resolution + i) * 
+                                                                                (angular_resolution / speed)), bodyState, false))
```
Comment (doudou) [inline base/samples/LaserScan.hpp]: This looks weird ... You call this "Interpolated" but are not interpolating the transform. I guess that you should make this a parameter of the method.

**Comment by goldhoorn** [comment]
@TreNo: can you please update the PR?

**Comment by doudou** [comment]
Closing this due to inactivity. Reopen if needed.

**Pull request closed without merging**